.changeset/chatty-dingos-sneeze.md

```diff
@@ -0,0 +1,2 @@
+---
+---
```

Comment:
Is this intended?

Reply:
Yeah, this won't trigger a release of this package, but I guess we should. Let me replace it.